Themis.nsi

```diff
@@ -0,0 +1,35 @@
+Name "Themis"
+OutFile "build\InstallThemis.exe"
+LicenseData "LICENSE"
+InstallDir "$PROGRAMFILES\Themis"
+
+VIAddVersionKey "ProductName"     "Themis"
+VIAddVersionKey "CompanyName"     "Cossack Labs Limited"
+VIAddVersionKey "LegalCopyright"  "(c) Cossack Labs Limited"
+VIAddVersionKey "FileDescription" "Themis library installer"
+VIAddVersionKey "FileVersion"     "0.11.2"
```

Comment:
we need add to our checklist for releases where we will need to update version. maybe place here some placeholder and set correct version before using this file?